src/component/Calculator/CalculatorComponent.js

```diff
@@ -0,0 +1,170 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] I noticed that you disable **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: